aii-core/src/main/perl/Shellfe.pm

```diff
@@ -30,8 +30,10 @@ use File::Path qw(mkpath rmtree);
 use File::Basename qw(basename);
 use DB_File;
 use Readonly;
-
+use Parallel::ForkManager 0.7.6;
 our $profiles_info = undef;
+use AII::DHCP;
+use 5.10.1;
```

Comment:
@kwaegema also update the description with was is deprecated.
maybe also mention the change in dhcp update (afaik, dhcp is now also updated before remove; that might also be meaningful)